start-site/src/main/resources/application.yml

```diff
@@ -1427,6 +1424,181 @@ initializr:
               href: https://googlecloudplatform.github.io/spring-cloud-gcp/reference/html/index.html#cloud-storage
             - rel: guide
               href: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample
+    - name: AI
+      content:
+        - name: Spring AI for Azure OpenAI
+          id: spring-ai-azure-openai
+          compatibility-range: "[3.1.0,3.3.0-M1)"
+          group-id: org.springframework.ai
+          artifact-id: spring-ai-azure-openai-spring-boot-starter
+          description: Azure’s OpenAI offering, powered by ChatGPT, extends beyond traditional OpenAI capabilities, delivering AI-driven text generation with enhanced functionality.
+          bom: spring-ai
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-ai/reference/api/clients/azure-openai-chat.html
+        - name: Spring AI vector database for Azure
+          id: spring-ai-vectordb-azure
+          compatibility-range: "[3.1.0,3.3.0-M1)"
+          group-id: org.springframework.ai
+          artifact-id: spring-ai-azure-vector-store-spring-boot-starter
+          description: Azure AI Search is a versatile cloud-hosted cloud information retrieval system that is part of Microsoft’s larger AI platform. Among other features, it allows users to query information using vector-based storage and retrieval.
```

Comment:
`cloud-hosted cloud information retrieval system` reads a little strangely to me. Perhaps `cloud-hosted information retrieval system` instead?

Reply:
I think I copied it from their website, which was https://azure.microsoft.com/en-us/products/ai-services/cognitive-search and is now https://azure.microsoft.com/en-us/products/ai-services/ai-search/   

`Azure AI Search, an AI-powered information retrieval platform.` is the first intro on that web page, so we can go with that.